contracts/nft/erc721m/ERC721MInitializableV1_0_1.sol

```diff
@@ -399,6 +423,13 @@ contract ERC721MInitializableV1_0_0 is
         emit SetContractURI(uri);
     }
 
+    /// @notice Sets the frozen status
+    /// @param frozen The frozen status to set
+    function setFrozen(bool frozen) external onlyOwner {
```

Comment:
Is this standard? Looks like we have `setTransferable` for 1155. Do we need to consolidate?

Reply:
Yeah let's standardize the naming of this flag. Could lead to issues as the true false states for `frozen` and `transferrable` are opposites.